Vignette of workflow and refactor of functions name

davidecrs opened the pull request:
Created vignette for basic workflow of `spatialDE`.

The following functions name have been updated:
- `run_spatialDE` to `run`
- `run_model_search` to `model_search`

Function usage on tests and vignette have been updated as well.

davidecrs committed:
Initial structure of spatialDE workflow

davidecrs committed:
Change filename from spatialDEr.Rmd to spatialDE.Rmd

davidecrs committed:
Updated structure of vignette

davidecrs committed:
Updated .Rbuildignore

davidecrs committed:
Updated .gitignore

davidecrs committed:
Move workflow vignette to spatialDE markdown

davidecrs committed:
Updated vignette

davidecrs committed:
spatialDE run on test genes

davidecrs committed:
Changed model_search name function

davidecrs committed:
Changed function name of spatialDE run and model_search

davidecrs committed:
Updated package documentation

davidecrs committed:
Updated test and vignette function

davidecrs committed:
Updated vignette with author info

davidecrs committed:
Updated vignette

davidecrs committed:
Comment on ggplot lines

milanmlft reviewed (commented):
Looking good! Once the plots work, we can merge.

davidecrs committed:
Added ggplot2 to DESCRIPTION suggests

davidecrs committed:
Increment version number

milanmlft committed:
Clean up vignette

- Add Introduction and Installation sections
- Update section headers
- Some reformatting

milanmlft committed:
Add sessionInfo to vignette

milanmlft committed:
Load data directly from URLs

milanmlft committed:
Use 'df_print: paged'

milanmlft committed:
Add some details for each step and sort results
